### ISSUE TITLE
RoboNeuro fails when converting checklist to issue

### ISSUE THREAD

**Issue by ltetrel** [neurolibre/roboneuro#15]
I converted one of the check to an issue:
https://github.com/neurolibre/neurolibre-reviews/issues/4

Weird output from RoboNeuro:
https://github.com/neurolibre/neurolibre-reviews/issues/5

**Comment by emdupre** [comment]
It's not going to be possible to open an issue on the target repo, unfortunately.

Instead, if we keep the checklists, we should pull in this behavior:

https://github.com/openjournals/joss-reviews/issues/3798#issuecomment-932663625

**Comment by ltetrel** [comment]
Can we keep the checklist but disable the issue creation (to not confuse the reviewer?)

**Comment by emdupre** [comment]
Unfortunately there's not (yet) a way to disable this, see :
https://github.com/github/feedback/discussions/4261

**Comment by ltetrel** [comment]
Hmm sad, then I would go for completely removing it, so no issues created at a bad place :)
wdyt ?

**Comment by emdupre** [comment]
I'm torn : I find the "% complete" incredibly helpful as an editor, and I wonder if we couldn't prevent some (hopefully significant) amount of confusion through better documentation. That being said, there will always be some number of testers who get confused. 

On balance, since we have a relatively small tester pool, though, maybe it's OK if folks are confused their first time ? vs JOSS who have a large and continuously rotating pool of reviewers, so the number of accidentally created issues can quickly pile up.

TL;DR : I'm leaning towards keeping it, but that's not a strong opinion !!

**Comment by ltetrel** [comment]
Actually I agree with you haha. That true that if we keep having the same reviewers that is not a huge deal since they will be aware of that.
What is still important though is to update the checklist to something that suits to us (another discussion not for this thread).
